ovos_plugin_common_play/ocp/__init__.py

```diff
@@ -42,11 +43,14 @@ class OCP(OVOSAbstractApplication):
     }
 
     def __init__(self, bus=None, lang=None, settings=None):
-        settings = settings or OCPSettings()
+        # settings = settings or OCPSettings()
         res_dir = join(dirname(__file__), "res")
         gui = OCPMediaPlayerGUI()
         super().__init__(skill_id="ovos.common_play", resources_dir=res_dir,
```

Comment:
here should also use OCP_ID; and it highlights an inconsistency, we had both ovos.common_play and ovos_common_play in use.... this is supposed to be the source of truth so `ovos.common_play` should be adopted in OCP_ID

Reply:
`ovos.common_play` or `ovos_common_play`? If we follow the convention used for skills, it would probably be `ovos_common_play.openvoiceos`....

Reply:
I guess `ovos.common_play` is the least likely to break things looking at test failures